### ISSUE TITLE
mantain alignment of multiline content when using format "simple"

### ISSUE THREAD

**Issue by mgacc0** [yihui/knitr#2021]
When using `format = "simple"` and some cells contain newline characters, would it be manageable to implement an option for mantaining visual alignment?

For example:
```r
library(tidyverse, knitr)
tibble(id = c(1,2,3), text = c("line1\nline2", "bbbb", "cccc")) %>%
    knitr::kable(format = "simple")
```

```
 id  text        
---  ------------
  1  line1
line2 
  2  bbbb        
  3  cccc        
```

What I intend to obtain is:
```
 id  text        
---  ------------
  1  line1
     line2 
  2  bbbb        
  3  cccc        
```

This option would be FALSE by default.

---

By filing an issue to this repo, I promise that

- [x] I have fully read the issue guide at https://yihui.org/issue/.
- [x] I have provided the necessary information about my issue.
    - If I'm asking a question, I have already asked it on Stack Overflow or RStudio Community, waited for at least 24 hours, and included a link to my question there.
    - If I'm filing a bug report, I have included a minimal, self-contained, and reproducible example, and have also included `xfun::session_info('knitr')`. I have upgraded all my packages to their latest versions (e.g., R, RStudio, and R packages), and also tried the development version: `remotes::install_github('yihui/knitr')`.
    - If I have posted the same issue elsewhere, I have also mentioned it in this issue.
- [x] I have learned the Github Markdown syntax, and formatted my issue correctly.

I understand that my issue may be closed if I don't fulfill my promises.

**Comment by yihui** [comment]
I don't think simple tables support newlines in cells. The new line will be a different row (if you don't mind that, we can consider the support). Only multiline tables and grid tables support newlines in cells: https://pandoc.org/MANUAL.html#tables

**Comment by mgacc0** [comment]
Exactly. I would find of interest that kable could generate output in format "multiline_table": https://pandoc.org/MANUAL.html#extension-multiline_tables

**Comment by yihui** [comment]
Yes, it will be nice to switch to a multiline table when `\n` is detected in the table cells (I remember someone has filed a feature request before). I don't have time to work on this myself, so I'll appreciate the contribution from someone else.

**Comment by yihui** [comment]
Implemented in #2501. `kable(format = "simple")` now emits a Pandoc multiline table when any cell contains a line break, so multiline content keeps its alignment instead of leaking into misaligned rows. Tables with no line breaks are unchanged.
